pkg/storage/wrapper/prefix/prefix_wrapper.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package prefix
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+)
+
+// package KIDPrefix offers provider.Store wrappers for stores with certain constraints on key values such as
+// reserved KIDPrefix values (eg key IDs starting with underscore) by prefixing key values.
+
+// KIDPrefix is the prefix added to KIDs in StorePrefixWrapper.
+const KIDPrefix = "p"
```

Comment:
This is a generic wrapper... why is it so key-focused?

Reply:
this wrapper only prefixes the keyID
the values are untouched

Reply:
@Baha-sk it's the "KID" term that associates this wrapper with the KMS.

Reply:
renamed to `ID`